### PR TITLE
fix: disk metrics error

### PR DESCRIPTION
**Pull request by fuxingZhang**
fix issue https://github.com/ClickHouse/clickhouse_exporter/issues/80

clickhouse_free_space_in_gigabytes and clickhouse_total_space_in_gigabytes  are removed.
Because they can be calculated by free_space_in_bytes and total_space_in_bytes.

Visit http://localhost:9116/metrics, you can see: 

image

image

### CODE REVIEW

**Review by Slach — CHANGES_REQUESTED** [review]

File: exporter/exporter.go [hunk 1]
```diff
@@ -181,18 +178,27 @@ func (e *Exporter) collect(ch chan<- prometheus.Metric) error {
...
-			prometheus.NewDesc(
-				namespace+"_"+metricName(dm.key),
-				"Disks "+dm.key+" capacity", []string{}, nil),
-			prometheus.CounterValue, float64(dm.value))
-		ch <- newMetric
+		newFreeSpaceMetric := prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Namespace: namespace,
+			Name:      "free_space_in_bytes",
+			Help:      "Disks free_space_in_bytes capacity",
+		}, []string{"name"}).WithLabelValues(dm.name)
```
Comment (Slach) [inline exporter/exporter.go:191]: instead of `name` better will `disk` label here

File: exporter/exporter.go [hunk 2]
```diff
@@ -181,18 +178,27 @@ func (e *Exporter) collect(ch chan<- prometheus.Metric) error {
...
+		newFreeSpaceMetric.Set(dm.freeSpace)
+		newFreeSpaceMetric.Collect(ch)
+
+		newTotalSpaceMetric := prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Namespace: namespace,
+			Name:      "total_space_in_bytes",
+			Help:      "Disks total_space_in_bytes capacity",
+		}, []string{"name"}).WithLabelValues(dm.name)
```
Comment (Slach) [inline exporter/exporter.go:199]: `disk` instead of `name` more mental

**Review by Slach — CHANGES_REQUESTED** [review]

File: exporter/exporter.go [hunk 3]
```diff
@@ -267,6 +273,48 @@ func (e *Exporter) parseKeyValueResponse(uri string) ([]lineResult, error) {
...
+	lines := strings.Split(string(data), "\n")
+	var results []diskResult = make([]diskResult, 0)
+
+	for i, line := range lines {
+		parts := strings.Fields(line)
+		if len(parts) == 0 {
+			continue
+		}
+		if len(parts) != 3 {
+			return nil, fmt.Errorf("parseKeyValueResponse: unexpected %d line: %s", i, line)
```
[the remark this answers is not in the document]
Comment (fuxingZhang) [reply exporter/exporter.go]: Thanks for the guidance, I have revised and resubmitted.

**Comment by fuxingZhang** [comment]
Thanks for the guidance, I have revised and resubmitted.

**Review by Slach — APPROVED** [review]

**Pull request merged**